crates/sargon/src/gateway_api/models/logic/response/transaction/status/transaction_status.rs

```diff
@@ -1,5 +1,89 @@
 use crate::prelude::*;
 
+#[cfg(test)]
+impl TransactionStatusResponse {
+    // Helper functions to create sample responses
+
+    pub fn sample_empty() -> TransactionStatusResponse {
```

Comment:
Prefer `Self` in return position and in construction.

And also, since you construct so many `TransactionStatusResponse` here, I would probably have used a ctor!

And for case of `error_message: None` you can use a specific ctor, which calls `new` called `success`?